src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/cardgrid/CardData.kt

```diff
@@ -0,0 +1,14 @@
+package org.wycliffeassociates.otter.jvm.app.ui.cardgrid
+
+import org.wycliffeassociates.otter.common.data.model.Collection
+import org.wycliffeassociates.otter.common.data.model.Content
+
+data class CardData(
+    val item: String,
+    val dataType: String,
+    val bodyText : String,
+    val sort: Int,
+    val contentSource: Content? =null,
+    val collectionSource: Collection? =null
```

Comment:
These cards don't need to know about the underlying content they represent.

What I mean by that is, they provide the appropriate places to set content to, and whatever configures it (binds content to the UI) can know about the actual domain stuff.

Like, The card cares about major text, minor text, button text/icon, graphic, whatever. It doesn't need to know that the major text is ACTUALLY a book title, it doesn't care, it just cares that you told it this is the text you want in the top slot.